subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyReportTaskIntegrationTest.groovy

```diff
@@ -1054,10 +1052,10 @@ compileClasspath - Compile classpath for source set 'main'.
         """
 
         when:
-        run ":a:dependencies"
+        fails ':a:dependencies'
 
         then:
-        !output.contains("\ncompile\n")
+        failure.hasErrorOutput("Dependencies can no longer be declared using the `compile` configuration.")
```

Comment:
Again, this test is testing the `deprecateFor` method and thus a warning should be emitted, with no error thrown.